.eslintrc.js

```diff
@@ -2,5 +2,13 @@ module.exports = {
   extends: 'frost-standard',
   globals: {
     capture: false
+  },
```

Comment:
Why were these added?  Did you encounter any issues when you didn't have them?  (Wouldn't have expected you too since you've upgraded `ember-test-utils` to `^8.1.0`)